docs/features/agic-reconcile.md

```diff
@@ -0,0 +1,21 @@
+## Reconcile senario
+When an Application Gateway is deployed through ARM template, a requirement is that the gateway configuration should contain a probe, listener, rule, backend pool and backend http setting. When such a template is re-deployed with minor changes (for example to WAF rules) on Gateway that is being controlled by AGIC, all the AGIC written rules are removed. Given such change on Application Gateway doesn’t trigger any events on AGIC, AGIC doesn’t reconcile the gateway back to the expected state. 
+
+## Solution
+To address the problem above, agic periodically checks if the latest gateway configure is different from what it cached, and reconcile if any to make gateway configure is eventual right.
```

Comment:
agic -> AGIC
configure -> configuration

suggestion: if needed to make gateway configuration is eventually correct.

Reply:
> agic -> AGIC
> configure -> configuration
> suggestion: if needed to make gateway configuration is eventually correct.

Fixed